src/components/main-view/main-view.jsx

```diff
@@ -1,77 +1,159 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import { Container, Row, Col, Button, Form, Card } from 'react-bootstrap';
+import { Container, Row, Col } from 'react-bootstrap';
+import { BrowserRouter as Router, Route, Routes, Navigate } from 'react-router-dom';
 import { MovieCard } from '../movie-card/movie-card';
+import { MovieView } from '../movie-view/movie-view';
 import { LoginView } from '../login-view/login-view';
 import { SignupView } from '../signup-view/signup-view';
+import { ProfileView } from '../profile-view/profile-view';
+import { NavigationBar } from '../navigation-bar/navigation-bar';
 
 export const MainView = () => {
   const [movies, setMovies] = useState([]);
```

Comment:
Make the following changes to your `MainView`:
-  This is the correct position.

![](https://coach-courses-us.s3.amazonaws.com/comments/users/40649/79b1cbc79e4742553543d1df21554334/Screenshot-2024-08-17-093942.png)